sui-execution/latest/sui-adapter/src/static_programmable_transactions/loading/translate.rs

```diff
@@ -72,6 +72,9 @@ fn input(env: &Env, arg: CallArg) -> Result<(L::InputArg, L::InputType), Executi
                 L::InputType::Fixed(ty),
             )
         }
+        CallArg::BalanceWithdraw(_) => {
+            todo!("Load balance withdraw call arg")
+        }
```

Comment:
As a heads up, we are actively in the process of trying to get this code up and running. So let's be mindful of that as the downstream effects of having to deal with this now rather than later might make @tzakian and I cry many tears

Reply:
I don't need this piece of code working right now. I only need the CallArg to be visible in a transaction so that I can test the scheduler logic.